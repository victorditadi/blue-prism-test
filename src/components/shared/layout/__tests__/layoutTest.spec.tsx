import { render } from '@testing-library/react';

import { Layout } from '..';

describe('Component: Layout', () => {
  it('should render Layout component', () => {
    expect(Layout).toMatchSnapshot();
  });

  it('should contain Header', () => {
    const { queryByTestId } = render(
      <Layout>
        <h1>Children</h1>
      </Layout>
    );
    expect(queryByTestId('header-title')).toBeTruthy();
  });

  it('should contain Children', () => {
    const { queryByTestId } = render(
      <Layout>
        <h1 data-testid="children">Children</h1>
      </Layout>
    );
    expect(queryByTestId('children')).toBeTruthy();
  });
});
