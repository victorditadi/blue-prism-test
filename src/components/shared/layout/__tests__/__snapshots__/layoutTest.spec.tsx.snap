// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Layout should render Layout component 1`] = `[Function]`;
